communication/slack-config/channels.yaml

```diff
@@ -389,9 +389,11 @@ channels:
   - name: ug-big-data
     id: C0ELB338T
   - name: ug-vmware
+  - name: undistro
   - name: velero-dev
   - name: velero-users
     id: C6VCGP4MT
+  - name: velero
```

Comment:
This isn't supposed to be here :)

Reply:
order fixed

Reply:
No, I mean that channel has been removed, your rebase got confused.

Reply:
now it's removed